src/main/java/com/bulkbuy/enterprise/dao/OrderDAO.java

```diff
@@ -1,4 +1,26 @@
 package com.bulkbuy.enterprise.dao;
 
-public class OrderDAO {
+import com.bulkbuy.enterprise.dto.Order;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class OrderDAO implements IOrderDAO{
+    List<Order> allOrders = new ArrayList<>();
+
+    @Override
+    public void create(Order order) {
+        try {
+            allOrders.add(order);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Use proper logging instead of printStackTrace().
- Loggins has the concept of priority (error, debug, warn, etc.), which printStackTrace doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with printStackTrace.